package/yast2-tune.spec

```diff
@@ -12,60 +12,61 @@
 # license that conforms to the Open Source Definition (Version 1.9)
 # published by the Open Source Initiative.
 
-# Please submit bugfixes or comments via http://bugs.opensuse.org/
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
 
 Name:           yast2-tune
-Version:        4.1.0
+Version:        4.2.0
 Release:        0
+Summary:        YaST2 - Hardware Tuning
+License:        GPL-2.0-or-later
+Group:          System/YaST
+Url:            https://github.com/yast/yast-tune
 
-BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Source0:        %{name}-%{version}.tar.bz2
 
 BuildRequires:  update-desktop-files
 BuildRequires:  yast2
-BuildRequires:  yast2-devtools >= 3.1.10
+BuildRequires:  yast2-devtools >= 4.2.2
+
 # hwinfo/classnames.ycp
 # Wizard::SetDesktopTitleAndIcon
 Requires:       yast2 >= 2.21.22
 Requires:       yast2-bootloader
+Requires:       yast2-ruby-bindings >= 1.0.0
 
-Provides:       yast2-config-hwinfo
-Provides:       yast2-trans-tune
-Provides:       yast2-tune-idedma
 Obsoletes:      yast2-config-hwinfo
 Obsoletes:      yast2-trans-tune
 Obsoletes:      yast2-tune-idedma
-Provides:       y2c_tune
-Provides:       y2t_tune
-Provides:       yast2-config-tune
-Provides:       yast2-trans-hwinfo
-Provides:       yast2-trans-idedma
 Obsoletes:      y2c_tune
 Obsoletes:      y2t_tune
 Obsoletes:      yast2-config-tune
 Obsoletes:      yast2-trans-hwinfo
 Obsoletes:      yast2-trans-idedma
 Obsoletes:      yast2-tune-devel-doc
 
-Requires:       yast2-ruby-bindings >= 1.0.0
-
-Summary:        YaST2 - Hardware Tuning
-License:        GPL-2.0-or-later
-Group:          System/YaST
+Provides:       yast2-config-hwinfo
```

Comment:
I think we can drop these ancient Obsoletes/Provides. They were needed like 15+ years ago...